node/src/components/linear_chain.rs

```diff
@@ -53,12 +53,6 @@ pub enum Event<I> {
         /// The deploys' execution results.
         execution_results: HashMap<DeployHash, ExecutionResult>,
     },
-    /// A continuation for `GetBlock` scenario.
-    GetBlockResult(BlockHash, Option<Box<Block>>, I),
-    /// A continuation for `BlockAtHeight` scenario.
-    GetBlockByHeightResult(u64, Option<Box<Block>>, I),
-    /// A continuation for `BlockAtHeightLocal` scenario.
-    GetBlockByHeightResultLocal(u64, Option<Box<Block>>, Responder<Option<Block>>),
```

Comment:
We never construct these events? So other components read directly from the storage? :grimacing:

Reply:
No, the _requests_ are being made. These `*Result` events were just used internally, sent from `LinearChain` to itself, instead of doing everything in an `async` block.